src/specs/components/site-header/demo/data--ec.js

```diff
@@ -28,6 +28,16 @@ module.exports = {
     code: 'en',
     overlay: {
       title: 'Select your language',
+      close: {
+        variant: 'ghost',
```

Comment:
not so important, but since we correctly hardcode this parameter in the template i would remove it from here.